core/ops_bin.rs

```diff
@@ -1,54 +1,23 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Comment:
Not sure I understand, I thought bin ops could be removed altogether now?

Reply:
@bartlomieju They can be removed but see my note in **Key Ideas** (top comment):
> Preserve concepts of json/bin/min as semantic groups of their inputs/outputs instead of their op-encoding strategy, preserving these groups will also facilitate partial transitions over to v8 Fast API for the "min" and "bin" groups

Also want to minimize changes to code outside of core, to make this PR easier to land/review.